packages/services/cdn-worker/src/handler.ts

```diff
@@ -29,102 +29,140 @@ type SchemaArtifact = {
 type ArtifactType = 'schema' | 'supergraph' | 'sdl' | 'metadata' | 'introspection';
 const artifactTypes = ['schema', 'supergraph', 'sdl', 'metadata', 'introspection'] as const;
 
-const createArtifactTypesHandlers = (
-  analytics: Analytics,
-): Record<
-  ArtifactType,
-  (targetId: string, artifactType: string, rawValue: string, etag: string) => Response
```

Comment:
this prevented me from clicking on `artifactTypesHandlers.supergraph` and going to definition.